sda-commons-dependencies/build.gradle

```diff
@@ -135,6 +135,7 @@ dependencies {
 
     // sda-commons-server-testing
     api "junit:junit:4.13"
```

Comment:
I wonder if we still need to set this here since it is also part of the dropwizard-bom(?)

Reply:
Also applies to assertj-core and mockito-core

Reply:
I'll give it a try and removed all test dependencies that are managed by Dropwizard.